Examples/SubscriberExample/Sources/Screens/Logging/SubscriberLogger.swift

```diff
@@ -3,10 +3,10 @@ import AblyAssetTrackingCore
 import Logging
 
 class SubscriberLogger: AblyAssetTrackingCore.LogHandler {
-    let swiftLog: Logger
+    private let swiftLog: Logger
```

Comment:
Is there a reason why this variable name takes the name of the language into its own name?

Reply:
Oh I see, that's from https://github.com/apple/swift-log, okay than)